code/modules/cargo/bounties/gardencook.dm

```diff
@@ -51,3 +44,22 @@
 	description = "Something sticky, something fun, ship us a honeybun."
 	reward = 3500
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/honeybun)
+
+/datum/bounty/item/chef/superbite
+	name = "Super Bite Burger" //Salt, Pepper, Boiled Egg, 5 Steak, 4 Tomato, 3 Cheese, Bacon, and a Bun. Should be more than 1800.
+	description = "Commander Tubbs thinks he can set a competitive eating world record. All he needs is a super bite burger shipped to him."
+	reward = 2750
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/burger/superbite)
+
+/datum/bounty/item/chef/poppypretzel
+	name = "Poppy Pretzel"
+	description = "Central Command needs a reason to fire their HR head. Send over a poppy pretzel to force a failed drug test."
+	reward = 1800
```

Comment:
You get these out of venders, 1000 is a better price

Reply:
You no longer get these from vendors.